var/spack/repos/builtin/packages/multitail/package.py

```diff
@@ -0,0 +1,55 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Multitail(MakefilePackage):
+    """MultiTail allows you to monitor logfiles and command output
+       in multiple windows in a terminal, colorize, filter and merge."""
+
+    homepage = "https://www.vanheusden.com/multitail/index.php"
+    url      = "https://www.vanheusden.com/multitail/multitail-6.4.2.tgz"
+
+    version('6.4.2', 'a0959f7b2385061080712afd8ae6e33d')
+
+    depends_on('ncurses')
+
+    def edit(self, spec, prefix):
+        makefile = FileFilter('Makefile')
+        nc_inc = spec['ncurses'].prefix.inc
+        nc_lib = spec['ncurses'].prefix.lib
+
+        # Use DESTDIR for the install because the Makefile doesn't
+        # consistently use PREFIX.
+        makefile.filter('DESTDIR=', 'DESTDIR={0}'.format(self.prefix))
+        makefile.filter('PREFIX=/usr', 'PREFIX=')
+        makefile.filter('CFLAGS\+=', 'CFLAGS+=-I{0} '.format(nc_inc))
```

Comment:
there is a same thing for includes, but I always forget...

Reply:
Looks like `cpp_flags`.  Sooo much I don't know..... 🙀

Reply:
Or maybe `include_flags`.  I'll keep reading through `lib/spack/llnl/util/filesystem.py`.

Reply:
`headers.include_flags`....